tutorials/evoked/30_eeg_erp.py

```diff
@@ -373,7 +374,260 @@
+# low-pass filter (see discussion in the above section
+# `Peak latency and amplitude`_)
+#
+# When using mean amplitude measures, selecting the time window based on
+# when the effect of interest (e.g., the difference between two conditions) can
+# inflate the likelihood of finding false positives in your results because
+# this approach is circular :footcite:`LuckGaspelin2017`. There are other, and
+# better, ways to identify a time window to use for extracting mean amplitude
+# measures. First, you can use *a priori* time window based on prior research.
+# A second way is to define a time window from an independent condition or set
+# of trials not used in the analysis (e.g., a "localizer"). A third approach is
+# to define a time window using the across-condition grand average. This latter
+# approach is not circular because the across-condition mean and condition
+# difference are independent of one another. The issues discussed above also
+# apply to selecting channels used for analysis.
+#
+# The following example demonstrates how to pull out the mean amplitude
+# from the left visual condition (i.e., the ``l_vis`` `~mne.Evoked` object)
+# using from selected channels and time windows. Stimulating the left visual
+# field is increases neural activity visual cortex of the contralateral
+# (i.e., right) hemisphere. We can test this by examining the amplitude of
+# the ERP for left visual field stimulation over right (contralateral) and
+# left (ipsilateral) channels. The channels used for this analysis are
+# ``eeg54`` and ``eeg57`` (left hemisphere), and ``eeg59`` and ``eeg55``
+# (right hemisphere). The time window used is .08 (``good_tmin``) to .12 s
+# (``good_tmax``) as it corresponds to when P100 typically occurs. The P100
+# is sensitive to left and right visual field stimulation. The mean amplitude
+# is extracted from the above four channels and stored in a
+# :class:`pandas.DataFrame`.
+
+# Select all of the channels and crop to the time window
+channels = ['eeg54', 'eeg57', 'eeg55', 'eeg59']
+hemisphere = ['left', 'left', 'right', 'right']
+l_vis_mean_roi = l_vis.copy().pick(channels).crop(
+    tmin=good_tmin, tmax=good_tmax)
+
+# Extract mean amplitude in µV over time
+mean_amp_roi = l_vis_mean_roi.data.mean(axis=1) * 1e6
+
+# Store the data in a data frame
+mean_amp_roi_df = pd.DataFrame({
+    'ch_name': l_vis_mean_roi.ch_names,
+    'hemisphere': ['left', 'left', 'right', 'right'],
+    'mean_amp': mean_amp_roi
+})
+
+# Print the data frame
+print(mean_amp_roi_df.groupby('hemisphere').mean())
+
+# %%
+# As demonstrated in the above example, the mean amplitude was higher and
+# positive in right compared to left hemisphere channels. It should be
+# reiterated that both that spatial and temporal window you use should be
+# determined in an independent manner (e.g., *a priori*, "localizer", from
+# and independent condition) and not based on the data you will use to test
```

Comment:
```suggestion
# determined in an independent manner (e.g., *a priori* from a "localizer" or
# other independent condition) and not based on the data you will use to test
```

Not sure I quite follow what you meant to say, but I think this is clearer.  If I've missed your intent, can you try clarifying the parenthetical?